locationTrivia-Objects/locationTrivia-Objects/FISTrivium.m

```diff
@@ -0,0 +1,24 @@
+//  FISTrivium.m
+
+
+#import "FISTrivium.h"
+
+@implementation FISTrivium
+
+- (instancetype)init {
+    self = [self initWithContent:@""
+                           likes:0   ];
```

Comment:
I'm really not a fan of this sort of indentation.. painful to maintain and not any more readable in my mind